Jenkinsfile.content

```diff
@@ -1,6 +1,23 @@
+@Library('va.gov-devops-jenkins-lib') _
+
 node('vetsgov-general-purpose') {
+  ref = params.ref
+  if (ref == "") {
+    ref = getLatestGitRef("vets-website", branch: 'master')
+  }
+
+  // check to see if build on master is running
+  is_running = jenkins.model.Jenkins.instance.getItem('testing/vets-website/master', jenkins.model.Jenkins.instance.getItem('testing')).lastBuild.building
+  if (is_running) {
+    slackSend(message:"Vetswebsite Content Build didn't build due to currently running vets-website build.",
+              color: "warning",
+              channel: "#vsp-something") // TODO update
```

Comment:
I think `cms-notifications` would work for this.

I think I would make the message say something like "Web content refresh on {buildtype} aborted due to currently running vets-website-build." I think that's more understandable to the cms team.